ArmPkg/Library/ArmPsciResetSystemLib/ArmPsciResetSystemLib.c

```diff
@@ -51,6 +52,7 @@ ResetCold (
 {
   ARM_MONITOR_ARGS  Args;
 
+  ZeroMem (&Args, sizeof (ARM_MONITOR_ARGS));

```

Comment:
If this is important, please do it in a separate patch.

Reply:
Okay, will update shortly.

Reply:
I ended up removing it. It was meant to prevent the garbage on the stack being penetrated to secure world. But given neither PSCI nor SMCCC clarifies this, the change does not seem as necessary.

Reply:
I added this to a separate commit.